includes/EmailNotification.php

```diff
@@ -458,30 +455,68 @@ private function emailExtensionEnabled() {
 	}
 
 	/**
-	 * When a page is created, the previousRevId is always 0.
-	 * @return bool
-	 */
-	private function isNewPage() {
-		return $this->previousRevId == 0;
-	}
-
-	/**
 	 * Send a watched page edit email using the new Email extension.
 	 * @param $user User
 	 */
 	private function sendUsingEmailExtension( \User $user ) {
-		F::app()->sendRequest( 'Email\Controller\WatchedPage', 'handle',
-			[
+
+		if ( $this->isArticlePageEdit() ) {
+			$controller = 'Email\Controller\WatchedPage';
+		} elseif ( $this->isArticleComment() ) {
+			$controller = 'Email\Controller\ArticleComment';
+		} elseif ( $this->isBlogComment() ) {
+			$controller = 'Email\Controller\BlogComment';
```

Comment:
Ok, I think this should be addressed by the changes made in the calling function.